internal/server/manage_important_information.go

```diff
@@ -27,52 +29,86 @@ type manageDeputyImportantInformationVars struct {
 	AnnualBillingInvoiceTypes []sirius.DeputyAnnualBillingInvoiceTypes
 	DeputyBooleanTypes        []sirius.DeputyBooleanTypes
 	DeputyReportSystemTypes   []sirius.DeputyReportSystemTypes
+	IsFinanceManager          bool
 }
 
 func renderTemplateForImportantInformation(client ManageProDeputyImportantInformation, defaultPATeam int, tmpl Template) Handler {
 	return func(perm sirius.PermissionSet, w http.ResponseWriter, r *http.Request) error {
-
 		ctx := getContext(r)
 		routeVars := mux.Vars(r)
 		deputyId, _ := strconv.Atoi(routeVars["id"])
 
-		deputyDetails, err := client.GetDeputyDetails(ctx, defaultPATeam, deputyId)
-		if err != nil {
-			return err
+		vars := manageDeputyImportantInformationVars{
+			Path:      r.URL.Path,
+			XSRFToken: ctx.XSRFToken,
+			DeputyId:  deputyId,
 		}
 
-		annualBillingInvoiceTypes, err := client.GetDeputyAnnualInvoiceBillingTypes(ctx)
-		if err != nil {
-			return err
-		}
+		group, groupCtx := errgroup.WithContext(ctx.Context)
```

Comment:
https://pkg.go.dev/golang.org/x/sync/errgroup 

We've been using this on `opg-sirius-lpa-frontend` where we have to make multiple requests and I think it's pretty neat, so included it here so you don't have 5 requests running in serial.

The only annoyance is using the correct `context.Context`. You want the requests running in `group` to use `groupCtx` so it can cancel them if one errors, but everywhere else you'll want the handler one in `ctx.Context`.